app/assets/javascripts/channels/books_ctrl.js

```diff
@@ -0,0 +1,38 @@
+(function(){
+  "use strict";
+
+  angular.module("app").controller("booksCtrl", function($scope, $http){
+    $scope.setup = function() {
+      getQuestions();
+    };
+
+    function getQuestions() {
+      $http.get('/api/v1/books.json').then(function(response){
+        $scope.questions = response.data;
+      });
+    }
+
+    $scope.tabs = [{
+            title: 'One',
+            url: 'one.tpl.html'
+        }, {
+            title: 'Two',
+            url: 'two.tpl.html'
+        }, {
+            title: 'Three',
+            url: 'three.tpl.html'
+    }];
+
+    $scope.currentTab = 'one.tpl.html';
+
+    $scope.onClickTab = function (tab) {
+        $scope.currentTab = tab.url;
+    }
+    
+    $scope.isActiveTab = function(tabUrl) {
+        return tabUrl == $scope.currentTab;
+    }
+
+    window.scope = $scope;
+  });
```

Comment:
This is the angular controller for the tabs on the book page. They weren't 100% working so that's why there's some commented out code in the show page.